feature-rich/package.json

```diff
@@ -72,5 +72,71 @@
     "build": "webpack --mode production",
     "build-dev": "webpack --mode development",
     "start": "webpack serve --open /index.html --mode development"
+  },
+  "resolutions": {
```

Comment:
Since you are adding explicit resolutions, can you add them consistently to all the demos? Here they are present only to few out of all the demos.

Reply:
Resolutions are needed beacause of [Wproofreader](https://github.com/ckeditor/ckeditor5/issues/13344)

Reply:
Without having it we'll fall into the same problem again once Wproofreader gets added to any other demo - and we won't know why. Let's be consistent across the configs, and ensure that `resolutions` are present.